app/templates/my-projects/project/recommendations/add.hbs

```diff
@@ -38,173 +38,33 @@
         {{#if (eq this.participantType "CB")}}
           <fieldset class="medium-margin-bottom">
             <legend><strong>Was a Quorum Present at your hearing?</strong></legend>
-            {{#if (and project.hearing.date project.hearing.location)}}
               <p class="text-small dark-gray">
                 Hearing occurred
-                on {{moment-format project.hearing.date "MM/D/YYYY"}}
-                at {{moment-format project.hearing.date "h:mm A"}}
+                on 11/3/2020
+                at 6:30 pm
```

Comment:
We're just hardcoding this for now and going to redesign templates for forms in a separate PR

Reply:
hmmm okay....

Reply:
@godfreyyeung  currently working on a branch that handles any changes to recommendations template. We figured we'd separate it out to avoid fitting everything into one PR. Do you think it's better to have done these template changes in the same PR as our model refactor?

Reply:
> We're just hardcoding this for now and going to redesign templates for forms in a separate PR

Gotcha. This needs to be made into a separate issue then. This issue needs to be closed and another issue opened to address this regression.

Ember mirage should allow us to mock whatever we need to deal with this sort of thing, but if it's more complicated than previously thought, definitely open up a new issue to address later. Thanks!

Reply:
Yes there is already an issue